### DISCUSSION TITLE
No module named 'telebot'

### DISCUSSION THREAD

**Discussion by tanwarjigyasu** [eternnoir/pyTelegramBotAPI#2110] [Q&A] [answered]
1. What version of pyTelegramBotAPI are you using?
4.14.1

3. What OS are you using?
Windows

5. What version of python are you using?
3.10.12

I get an saying ```No module named 'telebot'``` when I try to run ```import telebot```. I checked the ```AppData\Local\Programs\Python\Python39\Lib\site-packages``` while installing the package. It creates 2 new folders named "pyTelegramBotAPI-4.14.1.dist-info" and "telebot" and the main code of the package seems to be in "telebot" folder.  The "pyTelegramBotAPI" shows with the version specified when I run ```pip list```. My overall folder structure is like ```Some Folders/Bot Name/main.py``` and try to run the code in VS Code from the "Some Folders" in terminal. I tried reinstalling the package multiple times but it doen't help.

**Comment by Badiboy** [comment]
It works for all.  You set up something wrong. 
Not an issue, moved to discussions.

**Reply by tanwarjigyasu** [reply]
Then do you have any idea what I might be doing wrong ? I guess I am doing it alright but I will try reinstalling python if it still doesn't work after a restart

**Comment by Navy1989** [comment]
I also encounter this issue, my python version is 3.10.13, pyTelegramBotAPI version is 4.1.5.2

**Comment by coder2020official** [comment]
Check your virtual environment, check whether you properly installed it, check the import path..

**Comment by tanwarjigyasu** [comment, marked as answer]
Well turns out the problem was I had two versions of python installed in my device and my vs code was configured with the older one which doesn't work with pyTelegramBotAPI. It worked completely fine without any problems after changing it. Leaving this comment here in case someone has same trouble in future
